src/utilities.js

```diff
@@ -33,9 +33,12 @@ export function calcXpForLevel(lv) {
 
 var itemClasses = {};
 export async function getItemClass(itemName, scene) {
-    let itemClass = itemClasses[itemName];
-
-    return new itemClass.default(scene);
+    if (itemClasses.hasOwnProperty(itemName)) {
+        let itemClass = itemClasses[itemName];
+        return new itemClass.default(scene);
+    } else {
+        return null;
```

Comment:
I think we should log if we're passing in a bad item name here

Reply:
Updated! https://github.com/EthanVieira/rs-clicker/pull/79/commits/7291bf8df46a141986148de968f0b8284f32358a